cmd/publisher/commands/credentials.go

```diff
@@ -24,7 +25,7 @@ type CreateCredentialCommand struct {
 }
 
 func (cmd *CreateCredentialCommand) Run(args *cli_types.CommonArgs, ctx *cli_types.CLIContext) error {
-	cs := credentials.CredentialsService{}
+	cs := credentials.NewCredentialsService(logging.NewDiscardLogger())
```

Comment:
We are using a new `NewDiscardLogger()` to pass a no-op logging instance where we don't care too much about log output. (...at least for now)

Reply:
Why don't we care about logging? It's really the only diagnostic tool we have at this point if something goes wrong...?

Reply:
The credentials service being created here is for `cli` tooling. IIRC we are not prioritizing `cli` behavior, passing a no-op logger here to comply with the args without having to dig on the logging strategy for `cli` commands.

Reply:
Using verbosity arguments may be a better solution. But, I agree with @sagerb, its better to be overly verbose here instead of quiet.

aside; it would be really great to make the cli real

Reply:
The `CredentialService` is used to respond to existing API requests as well currently. All we wanted to do in this PR was to add an additional implementation that would not use the keychain for persistence. I'm not following you why we have to make this much of a change.